lib/setup.js

```diff
@@ -30,7 +30,7 @@ module.exports = function (options = {}) {
   let {
       app = express(),
       providers = [],
-      sessionStore,
+      sessionStore, // TODO: add deprecation notice
```

Comment:
@manuelurenah right below this declaration can you add a deprecation log (level `warn` that we'll be dropping support for the `sessionStore` property in Amphora v8?